src/__tests__/usePreviousDistinct.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { cleanup, renderHook } from 'react-hooks-testing-library';
```

Comment:
react-hooks-testing-library was updated to scoped version, so you need to import it from @testing-library/react-hooks

Reply:
Ah, haven't updated to latest in a bit. Will do so.